bindata/assets/alerts/cpu-utilization.yaml

```diff
@@ -10,18 +10,15 @@ spec:
         - alert: HighOverallControlPlaneCPU
           annotations:
             summary: >-
-              CPU utilization across all control plane nodes is more than 60% of the total available CPU. Control plane node outage may cause a cascading failure; increase available CPU.
+              CPU utilization across all three control plane nodes is higher than two control plane nodes can sustain; a single control plane node outage may
```

Comment:
@dgrisonnet are you ok with the changes ?

Reply:
yes it sounds fine to revert back to where it was before